index.js

```diff
@@ -0,0 +1,2 @@
+'use strict';
+module.exports = require('./lib/deepcrawl');
```

Comment:
At this point we might as well nuke the separate file and just put the actual code here. Any reason not to?